pkgs/development/tools/ocaml/ocamlformat/default.nix

```diff
@@ -25,11 +56,49 @@ with ocamlPackages; buildDunePackage rec {
     fix
     menhir
   ];
+in
+
+rec {
+  ocamlformat_0_11_0 = mkOCamlformat {
+    version = "0.11.0";
+    url = "https://github.com/ocaml-ppx/ocamlformat/archive/0.11.0.tar.gz";
```

Comment:
This URL is not good for `fetchurl` (the archive is dynamically generated and its hash may change).

Reply:
Are `fetchzip` et al. more robust because the hash is checked after unpacking?

Reply:
Precisely. `fetchzip` computes the hash of the contents after unpacking.

Reply:
Thanks ! I changed to fetchzip.
The added complexity (recursive argument set, unclear `sha256` argument) should go away when 0.11.0 is removed.